queries/snapshots/dashboard_subsidio_sppo_v2/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Changelog - snapshots/dashboard_subsidio_sppo_v2
 
+## [1.0.1] - 2025-05-08
+
+### Alterado
+
+- Altera snapshot `snapshot_sumario_faixa_servico_dia_pagamento.sql` (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/883)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)